ui/js/controllers/perf/dashboard.js

```diff
@@ -207,8 +208,8 @@ perf.controller('dashCtrl', [
 
         ThRepositoryModel.load().then(function () {
             $scope.projects = $rootScope.repos;
-            $scope.selectedRepo = _.find($scope.projects, {
-                name: $stateParams.repo ? $stateParams.repo : thDefaultRepo
+            $scope.selectedRepo = $scope.projects.find(function (project) {
```

Comment:
Would you mind changing these to arrow functions rather than using ``function`` here and the other places where you've added new code?  Thanks!

Reply:
Sure! One question - for the functions that use ternary expressions, after refactoring to arrow functions I'm getting a linter error "Arrow function used ambiguously with conditional expression." Should I refactor the ternary expression as well? Thanks!

Reply:
In some of those, if you simply wrap it in parentheses, then that takes care of it.  Though maybe there's a case where that doesn't work.  I'll try it out locally too.

Reply:
Thanks! Seems to work.